Data.TRAMS.Tests/Mappers/Response/TramsEstablishmentMapperTests.cs

```diff
@@ -47,7 +47,8 @@ public void GivenTramsAcademy_MapsToInternalAcademySuccessfully()
                     PersonalDevelopment = "3",
                     QualityOfEducation = "4",
                     ReligiousEthos = "Does not apply",
-                    SixthFormProvision = "1"
+                    SixthFormProvision = "1",
+                    WebLink = "fakelink.com"
```

Comment:
I'd probably use `http://example.com` for this as it's reserved, `http://fakelink.com` is an actual domain so I'd be wary even in examples of using this 😅

Reply:
Yeh good point - updated